shipment_management/public/js/custom_script.js

```diff
@@ -1,7 +1,12 @@
 frappe.ui.form.on("Delivery Note", {
```

Comment:
Might get confused with custom_script change 
in hooks.py name of this file is mentioned as custom_script.js 
![image](https://user-images.githubusercontent.com/80836439/125599449-ea08b04f-8480-4504-abbd-2d1c442597ee.png)